src/integrationTest/java/uk/gov/hmcts/divorce/common/SubmitAosIT.java

```diff
@@ -370,22 +365,11 @@ public void givenValidCaseDataWithDisputeWhenCallbackIsInvokedThenSendEmailToApp
         assertThatJson(actualResponse)
             .isEqualTo(jsonDocument.json());
 
-        verify(notificationService)
-            .sendEmail(eq(TEST_USER_EMAIL), eq(SOLE_APPLICANT_DISPUTED_AOS_SUBMITTED), anyMap(), eq(ENGLISH));
-
-        verify(notificationService)
-            .sendEmail(
-                eq(TEST_APPLICANT_2_USER_EMAIL),
-                eq(SOLE_RESPONDENT_DISPUTED_AOS_SUBMITTED),
-                argThat(allOf(hasEntry(PARTNER, "gwraig"))),
-                eq(WELSH)
-            );
-
-        verifyNoMoreInteractions(notificationService);
+        verifyNoInteractions(notificationService);
     }
 
     @Test
-    public void givenValidCaseDataForRespondentRepresentedWhenCallbackIsInvokedThenSendEmailToRespondentSolicitor() throws Exception {
+    public void givenValidCaseDataForRespondentRepresentedWhenCallbackIsInvokedThenSubmitAos() throws Exception {
```

Comment:
For these tests which have been renamed, could we include the Callback type?

...WhenAboutToSubmitCallbackIsInvoked... 

As we now have aboutToSubmit && submitted tests